### PR TITLE
Configure cross browser testing & Configure a visual report for the test executions

### PR DESCRIPTION
**Pull request by lorenacioata**
There is set up for both, Chrome and Firefox
A visual reporter is configured (e.g. Allure)
Test results are generated using configured reporter

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "The application to test is ToolsQA Website that can be found and accessed at: https://demoqa.com/",
   "main": "index.js",
   "scripts": {
-    "test": "wdio wdio.conf.js"
+    "test": "wdio wdio.conf.js",
+    "report": "npx generate allure --clean && allure open"
```
Comment (TaiaDimitrova) [inline package.json:8]: Update this command by adding the path to allure-results folder.

File: wdio.conf.js [hunk 2]
```diff
@@ -265,6 +275,26 @@ exports.config = {
...
+
+                if (exitCode !== 0) {
+                    return reject(reportError)
+                }
+
+                console.log('Allure report successfully generated')
+                resolve()
+            })
+        })
+    },
```
Comment (TaiaDimitrova) [inline wdio.conf.js:297]: I'd suggest to remove this one for now, to avoid getting failures on report generation.

**Review by TaiaDimitrova — CHANGES_REQUESTED** [review]
As part of this task is only configuration of the browsers and allure report, so please do not include the files that refers to tests.

File: package.json (hunk 1 above)
Comment (lorenacioata) [reply package.json:8]: I did after commit and the report still doesn't generate

**Review by TaiaDimitrova — APPROVED** [review]

**Pull request merged**